### ISSUE TITLE
AttributeError: 'DiscoveryV1' object has no attribute 'get_environments'

### ISSUE THREAD

**Issue by nisiya** [watson-developer-cloud/discovery-starter-kit#91]
Hi, I am getting this error when I try to start the python server.

**Comment by maniax89** [comment]
Sorry that it isn't working - to help me debug, could you please provide the following:

1. what commands you have run to get up to this point - if you are trying to develop locally, the following commands would be used in this order:
    - `npm --prefix client/knowledge_base_search install`
    - `npm --prefix client/knowledge_base_search run build`
    - `virtualenv .`
    - `source bin/activate`
    - `pip install -r server/python/requirements/dev.txt`
    - `python server/python/server.py`
1. where you are seeing the error message (local terminal window, in the browser, etc.)
1. what the error message/stack trace is

Hopefully we can resolve this given this information

**Comment by nisiya** [comment]
Thank you for responding. It seems that it was something I might have done wrong during the process. I downloaded the repository and started from scratch again and was able to get it to work. I did run into another error, that I was able to fix, but I just wanted to let you know about. When I ran `npm start`, I got this error below:
```
> REACT_APP_SERVER="http://localhost:5000" react-scripts start

'REACT_APP_SERVER' is not recognized as an internal or external command,
operable program or batch file.
```
I fixed it by changing the line in package.json:
`"start-js": "REACT_APP_SERVER=\"http://localhost:5000\" react-scripts start",`
to:
`"start-js": "react-scripts start REACT_APP_SERVER=\"http://localhost:5000\"",`
